examples/simple/main.tf

```diff
@@ -135,13 +135,14 @@ variable "materialize_instances" {
     memory_request          = string
     memory_limit            = string
     create_database         = optional(bool)
+    create_nlb              = optional(bool)
+    internal_nlb            = optional(bool)
```

Comment:
As per the discussion today, if we were to remove the Materialize CRs from Helm TF module, we should probably wait before merging this?

Reply:
I'm going to poke at separating them tomorrow to see if there is an easy path forward on that.

Reply:
As discussed in slack, I'm parking that CR move for now.